app/controllers/case_logs_controller.rb

```diff
@@ -116,6 +117,23 @@ def responses_for_page(page)
     end
   end
 
+  def get_inferred_answers(responses_for_page)
```

Comment:
I think this should probably go in the model since it's logic about how to handle data rather than request/response logic?

Reply:
Probably, I'll see how I can move that out

Reply:
Does it look better now?